tests/Metadata/TargetsMetadataTest.php

```diff
@@ -180,10 +180,6 @@ public function testGetDelegatedRoles(): void
                 self::assertTrue($delegatedRole->isKeyIdAcceptable($keyId));
             }
             self::assertFalse($delegatedRole->isKeyIdAcceptable('nobodys_key'));
-            foreach ($expectedRole['paths'] as $path) {
-                self::assertTrue($delegatedRole->matchesPath($path));
-            }
-            self::assertFalse($delegatedRole->matchesPath('/a/non/matching/path'));
```

Comment:
I think we should adapt this test coverage, not remove it. In other words, assert that `getDelegatedRoleForTarget()` returns the expected role(s) matching the target.

Reply:
We crossposted - I've just added a new test method that uses ::getDelegatedRolesForTarget().